README.md

```diff
@@ -1,25 +1,53 @@
 # Substrate Front End Template
 
-## Purpose
+This template allows you to create a front-end application that connects to a
+[Substrate](https://github.com/paritytech/substrate) node back-end with minimal
+configuration. To learn about Substrate itself, visit the
+[Substrate Developer Hub](https://substrate.dev).
 
-The purpose of this front-end template is to allow you to quickly create a
-front-end application that connects to a [Substrate](https://github.com/paritytech/substrate)
-node without much configuration and simplify the underlying connection and
-handling of [Polkadot-JS](https://polkadot.js.org/api/).
+The template is built with [Create React App](https://github.com/facebook/create-react-app)
+and [Polkadot js API](https://polkadot.js.org/api/). Familiarity with these tools
+will be helpful, but the template strives to be self-explanatory. To learn how
+this template was built, visit the
+[Substrate Front-End Tutorial](https://substrate.dev/docs/en/tutorials/substrate-front-end/).
 
-The encapsulation is handled by components inside `substrate-lib`. Substrate
-connection information is exposed out via `substrate-lib/useSubstrate.js`
-[custom hook](https://reactjs.org/docs/hooks-custom.html).
+## Using The Template
+
+### Installation
+
+The code can be installed using [git](https://git-scm.com/) and [yarn](https://yarnpkg.com/).
+```bash
+# Clone the repository
+git clone https://substrate-developer-hub/substrate-front-end-template.git
+cd ./substrate-front-end-template
+```
+
+## Usage
+You can start the template in development mode to connect to a locally running node
+```bash
+yarn start
+```
+
+You can also build the app in production mode,
+```bash
+yarn build
+```
+and open `build/index.html` in your favorite browser.
 
 ## Configuration
 
-The app configuration can be set in the `src/config` folder, with
-[common.json](./src/config/common.json)
-loaded in both development and production environments, and then the environment
-specific JSON file (with higher priority). More on React enviornment variables
-can be seen [here](https://create-react-app.dev/docs/adding-custom-environment-variables).
+The template's configuration is in three files in the `src/config` directory.
+* `common.json` affects both development and production environments.
+* `development.json` affects only the development environment with higher
+precedence than `common.json`.
+* `production.json` affects only the production environment with higher
+precedence than `common.json`.
+TODO @jimmychu what's `test.json`? Can you please write about it or remove it?
```

Comment:
`test.json` is used when running `yarn test` in the CI workflow.

Reply:
Thanks for adding the info :)